src/Agent.java

```diff
@@ -23,4 +25,38 @@ public void addProperty(Property property) {
     public void removeProperty(Property property) {
         this.properties.remove(property);
     }
+    
+    public static ArrayList<Agent> getAgentList() {
+    	return agentList;
+    }
+    
+    @Override
+    public String toString(){
+    	return this.getName();
+    }
+
+	@Override
+	public void call(Person person) {
+		if (this.getName() != person.getName()) {
+			System.out.println(this.getName() + " is calling " + person.getName() + " at " + person.getPhoneNo());
+		}
+		else {
+			System.out.println("Invalid operation, cannot call self");
+		}
+	}
+	
+	@Override
+	public void getInfo() {
```

Comment:
if you have a method name like getSmth -> you should return smth
In this case you can rename like `printInfo()`